### PR TITLE
Upgrade cuda drivers from 12.1 to 12.3

### PR DESCRIPTION
**Pull request by atheo89**
## Merge criteria:

- [ ] The commits are squashed in a cohesive manner and have meaningful messages.
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has manually tested the changes and verified that the changes work

### CODE REVIEW

File: cuda/ubi9-python-3.11/Dockerfile [hunk 1]
```diff
@@ -3,35 +3,35 @@ FROM ${BASE_IMAGE}
...
-      io.k8s.description="CUDA Python 3.11 builder image based on UBI8 for ODH notebooks" \
+      io.k8s.description="CUDA Python 3.11 builder image based on UBI9 for ODH notebooks" \
       authoritative-source-url="https://github.com/opendatahub-io/notebooks" \
       io.openshift.build.commit.ref="main" \
       io.openshift.build.source-location="https://github.com/opendatahub-io/notebooks/tree/main/cuda/ubi9-python-3.11" \
       io.openshift.build.image="quay.io/opendatahub/workbench-images:cuda-ubi9-python-3.11"
 
 # Install CUDA base from:
-# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/12.1.1/ubi9/base/Dockerfile
+# hhttps://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/12.3.2/ubi9/base/Dockerfile
```
Comment (jstourac) [inline cuda/ubi9-python-3.11/Dockerfile:15]: a nitpick optional
```suggestion
# https://gitlab.com/nvidia/container-images/cuda/-/blob/master/dist/12.3.2/ubi9/base/Dockerfile
```

**Comment by jstourac** [comment]
Has this been tested somehow?

**Comment by atheo89** [comment]
Closing this in favor: https://github.com/opendatahub-io/notebooks/pull/712

**Pull request closed without merging**